### DISCUSSION TITLE
Setting up local environment.

### DISCUSSION THREAD

**Discussion by Ann-design-max** [EddieHubCommunity/support#5454] [Q&A]
I need some help setting up my local environment to work properly for the repo good-first-issue-finder. I have followed the instructions to set up as shown in these images
![Good First Issue Finder - Google Chrome 23_09_2022 05_24_44](https://user-images.githubusercontent.com/62697124/191877095-6dfd2008-a436-4afe-a107-59b156ced468.png)
![Good First Issue Finder - Google Chrome 23_09_2022 05_24_55](https://user-images.githubusercontent.com/62697124/191877099-703cc598-d51f-43fe-954a-2f43f250d651.png)

but I'm getting this error when I try to login on local. 
![Good First Issue Finder - Google Chrome 23_09_2022 05_22_00](https://user-images.githubusercontent.com/62697124/191877221-936e7dd9-1271-4aa6-813d-73c35484445f.png)

Can anyone tell me what could be wrong please?

**Comment by ChinmayMhatre** [comment]
I think you might have made a mistake adding the auth token. You could try redoing the steps or try using the Gitpod. It requires no setup
